KSCrash.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = "KSCrash"
-  s.version      = "0.0.4"
+  s.version      = "0.0.5"
```

Comment:
it seems, you overwrite commit https://github.com/kstenerud/KSCrash/pull/99
Next version should 0.0.6